payments-core/src/main/java/com/stripe/android/GooglePayJsonFactory.kt

```diff
@@ -300,54 +299,76 @@ class GooglePayJsonFactory constructor(
         }
     }
 
-    /**
-     * [TransactionInfo](https://developers.google.com/pay/api/android/reference/request-objects#TransactionInfo)
-     */
     @Parcelize
-    data class TransactionInfo @JvmOverloads constructor(
-        /**
-         * ISO 4217 alphabetic currency code.
-         */
+    data class TransactionInfo internal constructor(
```

Comment:
Keeping this constructor internal for now. We should investigate if `GooglePayJsonFactory` and `TransactionInfo` actually need to be public or if we can make them internal and point to using `GooglePayLauncher` (see [this note](https://github.com/stripe/stripe-android/issues/3236#issuecomment-881117461)).